.gitignore

```diff
@@ -6,4 +6,7 @@
 /solana/frontend/node_modules
 
 .DS_Store
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
+
+/fio/lib/node_modules
```

Comment:
move this gitignore inside fio/lib. That way it's easier for someone to copy and use the package.

Reply:
Done